api/src/test/java/org/pmiops/workbench/api/CohortsControllerTest.java

```diff
@@ -426,6 +434,56 @@ public void testGetCohortsInWorkspace() throws Exception {
     assertThat(cohorts.size()).isEqualTo(2);
   }
 
+  @Test
+  public void testCreateCohortOwner() {
+    stubWorkspaceAccessLevelWithCreatorEmail(workspace, WorkspaceAccessLevel.OWNER);
+    Cohort saved =
+        cohortsController
+            .createCohort(workspace.getNamespace(), workspace.getId(), createDefaultCohort())
+            .getBody();
+    assertThat(saved.getCriteria()).isEqualTo(createDefaultCohort().getCriteria());
+    assertThat(saved.getCreator()).isEqualTo(CREATOR_EMAIL);
+    assertThat(saved.getCreationTime()).isNotNull();
+    assertThat(saved.getCreationTime()).isEqualTo(saved.getLastModifiedTime());
```

Comment:
Thank you, this is so much more readable.

Reply:
:-)